### DISCUSSION TITLE
Run formatter and linter (fixes) in a single command

### DISCUSSION THREAD

**Discussion by giampaolo** [astral-sh/ruff#8260] [General]
Sorry in advance if this has already been asked but I couldn't find anything. 
If I want to fix both linting errors and formatting, do I have to run 2 separate commands as in:

```
ruff format ...
ruff  --fix ...
```

...? Can't I do it by using a single command?

**Comment by giampaolo** [comment]
Ouch! It's already tracked here: https://github.com/astral-sh/ruff/issues/8232. Feel free to delete this issue and sorry for the noise.

**Reply by charliermarsh** [reply]
No worries -- we've heard this a bunch, working on it :)

**Reply by charliermarsh** [reply]
For what it's worth, we recommend running `ruff check --fix` followed by `ruff format`.

**Reply by giampaolo** [reply]
> For what it's worth, we recommend running ruff check --fix followed by ruff format.

This is good to know because I would have done the opposite.
